aicsimageio/dimensions.py

```diff
@@ -121,3 +123,17 @@ def __str__(self) -> str:
 
     def __repr__(self) -> str:
         return str(self)
+
+    def __getitem__(self, key: Union[str, Sequence[str]]) -> Union[int, Tuple]:
+        if isinstance(key, str):
+            return getattr(self, key)
```

Comment:
One design question here:
Should this return a tuple of length 1 instead? or be just a number?

Reply:
Oh great question. I would always return a Tuple

Reply:
done